server/api.js

```diff
@@ -11,9 +11,11 @@ router.get("/", (_, res) => {
 	res.json({ message: "Hello, world!" });
 });
 
+// I have edited this code so I could have access to ratings
 router.get("/energizers", async (_, res) => {
-	logger.debug("Welcoming everyone...");
-	const query = "SELECT * FROM energizers";
+	const query = `SELECT energizers.*, energizer_ratings.rating, energizer_ratings.id AS rating_id
+	FROM energizers
+	LEFT JOIN energizer_ratings ON energizers.id = energizer_ratings.energizer_id`;
```

Comment:
What happens with this query when there is multiple ratings?

Also as mentioned on @RobCso original code, using * and returning all columns is not good practice since the output of this endpoint will change if the database schema is changed.